### PR TITLE
First homework for typescript

### PR DESCRIPTION
**Pull request by KaterynaShtets**

### CODE REVIEW

File: src/shtets.kateryna/homework.ts [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+}
+
+/*
+  Create a function with two arguments that will return a list of length (n) with multiples of (x).
+
+  Assume both the given number and the number of times to count will be positive numbers greater than 0.
+*/
+export function countBy(n: number, x: number): number[]{
+  let res: number[] = [];
+  for(let i = 1; i <= x; i++ ){
```
Comment (denys-lymarenko-listware) [inline src/shtets.kateryna/homework.ts:69]: check if tslint plugin installed to your editor - different formatting (whitespaces)

`for (le`
`for(let i`

**Comment by denys-lymarenko-listware** [comment]
great, thanks for PR

- covered by unit tests
- good ts typing (types)


left small comment

**Comment by LimarenkoDenis** [comment]
great, thanks for PR

- covered by unit tests
- good ts typing (types)

left small comment

File: src/shtets.kateryna/homework.ts (hunk 1 above)
Comment (LimarenkoDenis) [inline src/shtets.kateryna/homework.ts:69]: check if tslint plugin installed to your editor - different formatting (whitespaces)

`for (le`
`for(let i`

**Pull request merged**